includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -39,7 +38,6 @@ class WC_Stripe_UPE_Payment_Gateway extends WC_Gateway_Stripe {
 		WC_Stripe_Payment_Methods::OXXO              => WC_Stripe_UPE_Payment_Method_Oxxo::class,
 		WC_Stripe_Payment_Methods::SEPA              => WC_Stripe_UPE_Payment_Method_Sepa::class,
 		WC_Stripe_Payment_Methods::P24               => WC_Stripe_UPE_Payment_Method_P24::class,
-		WC_Stripe_Payment_Methods::SOFORT            => WC_Stripe_UPE_Payment_Method_Sofort::class,
```

Comment:
This removal is causing unit test failures, and I am not sure how it's related to fixing the original issue.
At the very least we need to fix the unit test failures, but it feels like this should be something tackled by a different PR.

Reply:
Reverted the cleanup in [51e1cc3](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4252/commits/51e1cc32305e5dc20a6b9223a73ead8cb00f7a0e)